presto-parquet/src/main/java/com/facebook/presto/parquet/batchreader/decoders/rle/Int64ShortDecimalRLEDictionaryValuesDecoder.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.parquet.batchreader.decoders.rle;
+
+import com.facebook.presto.parquet.batchreader.decoders.ValuesDecoder.ShortDecimalValuesDecoder;
+import com.facebook.presto.parquet.dictionary.LongDictionary;
+
+import java.io.InputStream;
+
+public class Int64ShortDecimalRLEDictionaryValuesDecoder
```

Comment:
This class is not doing anything different than Int64RLEDictionaryValuesDecoder. Can we just use Int64RLEDictionaryValuesDecoder directly?

Reply:
I've updated this to add `ShortDecimalValuesDecoder` to the `implements` list on `Int64RLEDictionaryValuesDecoder`  instead. I personally prefer having the explicit reader class as it makes the code a bit easier to follow IMO